com.tracktion.Waveform13.yml

```diff
@@ -0,0 +1,146 @@
+      - --enable-libvpx
+      - --enable-libvorbis
+      - --enable-libx264
+    cleanup:
+      - /share/ffmpeg
+      - /include
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-5.1.6.tar.xz
+        sha256: f4fa066278f7a47feab316fef905f4db0d5e9b589451949740f83972b30901bd
+
+  - name: xjadeo
+    cleanup:
+      - /share/man
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/xjadeo/files/xjadeo/v0.8.14/xjadeo-0.8.14.tar.gz
+        sha256: 8f4b0cc7389b1466ce8a9219dd27404b5cbf49590620193930b242a92533d9dd
+
+      # - type: patch
+      #   path: patches/xjadeo-c-fix.patch
+
+  - name: xprop
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.4.tar.bz2
+        sha256: 8c77fb096e46c60032b7e2bde9492c3ffcc18734f50b395085a5f10bfd3cf753
+
+  - name: libusb
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-1.0.27/libusb-1.0.27.tar.bz2
+        sha256:  ffaa41d741a8a3bee244ac8e54a72ea05bf2879663c098c82fc5757853441575
+
+  - name: waveform
+    buildsystem: simple
+    build-commands:
+      # Icons
+      - install -Dm644 waveform13.png /app/share/icons/hicolor/64x64/apps/waveform13.png
+      - install -Dm644 waveform13.png /app/share/icons/hicolor/128x128/apps/waveform13.png
+      - install -Dm644 waveform13.png /app/share/icons/hicolor/256x256/apps/waveform13.png
+      - install -Dm644 waveform13.png /app/share/icons/hicolor/512x512/apps/waveform13.png
+
+      - install -Dm755 /usr/bin/desktop-file-edit -t /app/bin/
+      - install -Dm755 /usr/bin/ar -t /app/bin/
+      - install -Dm755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t /app/lib/
+      - install -Dm755 /usr/lib/$(gcc -print-multiarch)/libsframe.so.1 -t /app/lib/
+      - install -Dm644 com.tracktion.Waveform13.appdata.xml /app/share/appdata/com.tracktion.Waveform13.appdata.xml
+      - install -Dm755 apply_extra.sh /app/bin/apply_extra
+      - install -Dm755 waveform.sh /app/bin/waveform
+    post-install:
+      - install -d /app/extensions/Plugins
+    sources:
+      - type: extra-data
+        filename: waveform13.deb
+        only-arches:
+          - x86_64
+        url: https://cdn.tracktion.com/file/tracktiondownload/w13/1320/waveform13_13.2.0_amd64.deb
+        sha256: eb84aa44eb05026c81d09eb290e97261bdde5be286c10406d80bc8a5f11d7854
+        size: 178803508
+
+      - type: extra-data
+        filename: tracktion-download-manager.deb
+        only-arches:
+          - x86_64
+        url: https://cdn.tracktion.com/file/tracktiondownload/download-mgr/153/tracktion_download_manager_v1.5.3.deb
+        sha256: 577b8480c9d33a8b5cc213c61e9c3ff90a6e6a0a00efa0cbe7f4c2ec4f99cfa7
+        size: 3422932
+
+      - type: file
+        path: apply_extra.sh
+
+      - type: script
+        dest-filename: waveform.sh
+        commands:
+          - exec /app/extra/waveform13 $@
```

Comment:
```suggestion
          - exec /app/extra/waveform13 "$@"
```

Reply:
This would cause Waveform to treat all arguments as a singular, large argument. This is not what we want.

Reply:
definitely not. Re-read the manual.

Reply:
Fixed in latest commit